proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -143,6 +143,9 @@ message QueryAllBalancesRequest {
 
   // pagination defines an optional pagination for the request.
   cosmos.base.query.v1beta1.PageRequest pagination = 2;
+
+  // ibc_resolve is the flag to resolve the ibc hash denom into a human-readable form.
+  bool ibc_resolve = 3;
```

Comment:
should this be resolve denom? I can see on some chains they may want to use this as well for non ibc chains